ReactWindows/ReactNative.Shared/Reflection/EnumHelpers.cs

```diff
@@ -39,6 +32,22 @@ public static T Parse<T>(string value)
             return Parse<T>(value);
         }
 
+        public static bool TryParse<T>(string value, out T result)
+        {
+            var lookup = s_enumCache.GetOrAdd(
```

Comment:
This has the side effect of adding the element. I would reflect it in the name of the method?